cdap-ui/app/cdap/components/HttpExecutor/RequestHistoryTab/index.tsx

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import withStyles, { StyleRules, WithStyles } from '@material-ui/core/styles/withStyles';
+import classnames from 'classnames';
+import { LEFT_PANEL_WIDTH } from 'components/HttpExecutor';
+import HttpExecutorActions from 'components/HttpExecutor/store/HttpExecutorActions';
+import HttpExecutorStore from 'components/HttpExecutor/store/HttpExecutorStore';
+import { List } from 'immutable';
+import * as React from 'react';
+import { connect } from 'react-redux';
+
+const styles = (theme): StyleRules => {
+  return {
+    root: {
+      borderRight: `1px solid ${theme.palette.grey[300]}`,
+      height: '100%',
+    },
+    requestRow: {
+      padding: '10px',
+      lineHeight: '24px',
+      display: 'grid',
+      width: '100%',
+      gridTemplateColumns: '50px 1fr',
+      cursor: 'pointer',
+
+      '&:hover': {
+        backgroundColor: theme.palette.grey[700],
+      },
+    },
+    requestMethod: {
+      paddingLeft: '5px',
+      color: theme.palette.white[50],
+      width: '100%',
+      height: '100%',
+      fontWeight: 600,
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'flex-start',
+      fontSize: '10px',
+    },
+    requestMethodText: {
+      width: '100%',
+      textAlign: 'left',
+      alignSelf: 'center',
+    },
+    requestPath: {
+      width: `${LEFT_PANEL_WIDTH / 1.5}px`,
```

Comment:
why dividing by 1.5? i see you already have the gridTemplateColumns, can the width be controlled there instead?